src/mpi/romio/adio/include/adioi.h

```diff
@@ -158,6 +161,9 @@ typedef struct ADIOI_AIO_req_str {
     /* should probably make this a union */
 #ifdef ROMIO_HAVE_WORKING_AIO
     struct aiocb *aiocbp;
+#ifdef ROMIO_QUOBYTEFS
```

Comment:
Should this addition be tied to `ROMIO_HAVE_WORKING_AIO`? I was compile testing this using the libquobyte dev library and ran into a compile error because the working aio test failed on my machine.

Reply:
You are correct, it should not be tied to ROMIO_HAVE_WORKING_AIO. I will fix it and push it.